File: python/taskA.cpp
#include <iostream>
#include <string>

using namespace std;

int main(){
	string s;
	int curr, need;
	cin >> curr >> need >> s;
	if(s == "auto") cout << need;
    else if(s == "heat") {
        if(curr > need) cout << curr;
        else cout << need;
    }
    else if (s == "freeze") {
        if (curr < need) cout << curr;
        else cout << need;
    }
    else cout << curr;
	return 0;
}

File: python/taskA.py
troom = int(input())
tcond = int(input())

command = input()

if command == "auto":
	print(tcond)
elif command == "freeze":
	if troom < tcond:
		print(troom)
	else:
		print(tcond)
elif command == "heat":
    if troom > tcond:
        print(troom)
    else:
        print(tcond)
else:
    print(troom)
    
